### DISCUSSION TITLE
Can whisper give timestamps for every single word instead of every 5-10 words?

### DISCUSSION THREAD

**Discussion by yeetus1992** [openai/whisper#1002] [Q&A]
I use whisper to generate subtitles, so to transcribe audio and it gives me the variables „start“, „end“ and „text“ (inbetween start and end) for every 5-10 words. Is it possible to get these values for every single word? Do I have to like, use a different whisper model or similair? I would use that data to generate faster changing subititles.

Would be grateful for any help!

Sincereley

**Comment by glangford** [comment]
See
* https://github.com/openai/whisper/discussions/26
* https://github.com/openai/whisper/discussions/3

Work to create word level timestamps is in progress here, in a separate branch.
https://github.com/openai/whisper/tree/word-level-timestamps

**Reply by yeetus1992** [reply]
The first answer tells me to change whisper source code, yet every time i try it gives me erno 13 permission denied. 
`[Errno 13] Permission denied: '/Library/Python/3.9/site-packages/whisper/decoding.py/' `

How would you go about fixing this?

**Reply by slavovthinks** [reply]
@yeetus1992 https://askubuntu.com/questions/402980/give-user-write-access-to-folder
